subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -225,6 +226,25 @@ abstract class InHouseInstrumentationTestRunner :
         return super.onException(obj, e)
     }
 
+    @CallSuper
+    open fun initUITestConfig() {
```

Comment:
It allows `ScreenshotTestRunner` in main Avito project to disable clicks visualization

Reply:
Have a look at `subprojects/android-test/test-report/src/main/kotlin/com/avito/android/test/report/PerformanceTestRule.kt`. This rule is able to restore default value easily and without saving original value on its side.

Reply:
Is is possible with `UITestConfig.visualizeClicks = false` only, right?
if so, i think its too much for this PR to change inhouse runner interface

Reply:
I think that derived classes should be able to setup UITestConfig.

Reply:
This should be clear and obvious for derived classed where they can setup UITestConfig. If someone sets its values in config before super class initializes it then custom values of the derived class will be overwritten.

Reply:
it's ok for me, just have an issue with the weird scope of the PR


We should reconsider this UITestConfig singleton at all, Kaspresso made it right with its config imo